src/Relude/Nub.hs

```diff
@@ -33,14 +33,21 @@ module Relude.Nub
        , ordNub
        , sortNub
        , unstableNub
+       , foldNub
+       , hashFoldNub
+       , slowFoldNub
+       , foldNub'
+       , hashFoldNub'
+       , slowFoldNub'
```

Comment:
Could you move these functions to new `Relude.Extra.Nub` module? The imports will be cleaner since you can just write `import Relude`. But the idea is that these functions are not conventional and since `Relude.Nub` is exported by default, it could be a possible breaking change. Though, names are not ambiguous, so maybe it's okay to leave them here. What are you thoughts on this?

Reply:
Well, the only case in which this causes breaking changes is if someone has code that already uses these names. I have done a search on GH for these names and found no such instances. 

I think that a small risk of causing someone a breaking change is preferable to adding complexity to the module namespace, but that's not a very strong opinion, so happy to do it that way too.

Reply:
I didn't perform GitHub search. If there're no such functions even on Github, then it's okay to keep in this module. Usually, there're tradeoff between collisions, usefulness, naming and controversial implementation. But if there're no such functions and if we rename them to `foldToSet` and `foldToHashSet` and keep only `foldl'` implementation, then the usefulness of those functions become extremely high and without collisions we can keep them publicly accessible :+1: